auth0_flutter/android/src/main/kotlin/com/auth0/auth0_flutter/Auth0FlutterPlugin.kt

```diff
@@ -32,6 +32,10 @@ class Auth0FlutterPlugin: FlutterPlugin, MethodCallHandler, ActivityAware {
     LoginApiRequestHandler(),
     LoginWithOtpApiRequestHandler(),
     MultifactorChallengeApiRequestHandler(),
+    EmailPasswordlessLoginRequestHandler(),
+    PhoneNumberPasswordlessLoginRequestHandler(),
+    LoginWithEmailCodeRequestHandler(),
+    LoginWithSMSCodeRequestHandler(),
```

Comment:
Following the existing pattern, these should probably be called:
- EmailPasswordlessApiRequestHandler
- PhoneNumberPasswordlessApiRequestHandler
- LoginWithEmailCodeApiRequestHandler
- LoginWithSMSCodeApiRequestHandler